tdrs-backend/.env.example

```diff
@@ -0,0 +1,49 @@
+### Local Development Environment
+# Copy this file to `.env` and replace variables as needed
+#
+
+DISABLE_COLLECTSTATIC=1
+
+# Private JWT Key used to generate the client assertion
+JWT_KEY=a_secret_key
+
+#Base64 encoded public cert used for CI/CD pipeline testing
+JWT_CERT_TEST=a_public_cert
+
+# Add secret key from "Development Credentials" document
+SECRET_KEY=secret
+DJANGO_SECRET_KEY=secret
+
+# Parameters defined in the login.gov documentation https://developers.login.gov/oidc/
+CLIENT_ID=urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-dev
+ACR_VALUES=http://idmanagement.gov/ns/assurance/ial/1
+OIDC_OP_ISSUER=https://idp.int.identitysandbox.gov/
+OIDC_OP_JWKS_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/certs
+OIDC_OP_AUTHORIZATION_ENDPOINT=https://idp.int.identitysandbox.gov/openid_connect/authorize
+OIDC_OP_LOGOUT_ENDPOINT=https://idp.int.identitysandbox.gov/openid_connect/logout
+OIDC_OP_USER_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/userinfo
+CLIENT_ASSERTION_TYPE=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+
+# Override this endpoint as defined in "Development Credentials" document
+OIDC_OP_TOKEN_ENDPOINT='https://idp.int.identitysandbox.gov/api/openid_connect/token'
+
+# Add OIDC signing key as defined in "Development Credentials" document
+OIDC_RP_IDP_SIGN_KEY=a_secret_key
```

Comment:
This is a new ENV variable, it would be helpful to review what this is doing and why it's being introduced in this PR.

Reply:
Good catch, looks like this isn't actually used anywhere so it can be removed.

Reply:
@jtwillis92 This isn't the only variable that looks like it's being introduced as a new environment variable in this PR, I also see:

```
# Add OIDC Client ID from "Development Credentials" document
OIDC_RP_CLIENT_ID=client_id

# Add Auth Client ID from "Development Credentials" document
AUTH_CLIENT_ID=client_id
```

Should these also be removed? 

Could you say a little bit about where you found these variables & why you added them?

Reply:
It looks like `OIDC_RP_CLIENT_ID` is still used in `deploy-cloudgov-docker.sh` but holds a duplicate value of `CLIENT_ID`. The `AUTH_CLIENT_ID` is also a duplicate value but appears unused to me as well.

For context, I got a bash script with some exports for environment variables that @carltonsmith was using locally and found some `.env.local` and `.env.development` files committed that had some variables defined and I combined those all into one file for `.env.example`. It does appear this approach led to some duplicates that I missed so I'm glad you caught this!

Reply:
Great!